### DISCUSSION TITLE
Combine OGI and OGI_FU?

### DISCUSSION THREAD

**Discussion by tarcadius** [IM3S/LDAR_Sim#98] [Ideas]
We've talked about doing this because these two modules are so redundant and because OGI_FU is likely not a real company - probably actually the same company as is doing regulatory OGI surveys.

Can anyone think of a good reason not to do this? @tbarchyn @soroushojagh @AkaMrmuffin @KeeganShaw-GIS @c-vollrath

**Comment by tbarchyn** [comment]
Definitely good idea! I like the idea of modifying all companies so they only have a list of sites that they are doing (not all the sites, a subset). That subset could be 'all the sites' in some cases, but in the case here they'd have their normal OGI work + flags for follow-up.

A more complex thing could be a follow up scheme that directly messes with the OGI crew plans and puts certain sites on a 'hot list' to get done ASAP, or just straight up short circuit the agent's agency and tell them they must go to certain sites right away, even if it is far away and not optimal. I like conflating all of this to perception of cost so there is one variable that OGI crews get to think about: money.

This type of follow-up scheme obviously is super and desirable as you detect an ultra emitter in grande prairie, and want a follow up to get that thing fixed, but your OGI crews are down in foremost pac manning their way around nicely to minimize driving time. That is a giant problem because you want your OGI crews up in Grande Prairie now, not in 3 months. There has to be a formal way to re-prioritize the perceptive cost for the OGI crews, dangle some money in front of them so they make more money driving up to GP to kill that ultra-emitter and they forget about Foremost. 

Calling in that emergency call and special repositioning fees obviously makes OGI more expensive, but from a program level it makes sense.

**Comment by tarcadius** [comment]
Love these ideas. Ok will look into this more..

Your example makes me wonder why an operator can't grab the SNOOP and go find that 100 kg/h leak that's probably busting eardrums...

**Comment by tarcadius** [comment]
@KeeganShaw-GIS  @AkaMrmuffin  was this done?

**Comment by KeeganShaw-GIS** [comment]
THey are the same module, but it uses an is_followup flag. So different crews are used for followup and for OGI. I have moved into discussion because the ideas are pretty complex .
